client/app/components/conversation.coffee

```diff
@@ -42,9 +42,6 @@ module.exports = React.createClass
             length = MessageStore.getConversationsLength().get conversationID
             selectedMailboxID ?= Object.keys(message.get('mailboxIDs'))[0]
 
-        displayConvs = AccountStore.hasConversationEnabled selectedMailboxID
```

Comment:
I think conversation have to be disabled in trashbox otherwise we get weird behaviour. Could you check ?